test/unit/org/apache/cassandra/io/sstable/SSTableLoaderTest.java

```diff
@@ -101,6 +101,8 @@ public void cleanup()
               We force a GC here to force buffer deallocation, and then try deleting the directory again.
               For more information, see: http://bugs.java.com/bugdatabase/view_bug.do?bug_id=4715154
               If this is not the problem, the exception will be rethrown anyway.
+
```

Comment:
I'd massage this message a bit to remove the "Windows does not allow" etc. etc. completely.